app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -511,42 +513,57 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         }
     }
 
-    private fun externalAppLinkClicked(appLinkCommand: Command.HandleExternalAppLink) {
-        context?.let {
+    private fun openExternalDialog(intent: Intent, context: Context, fallbackUrl: String? = null, showFirstOption: Boolean = true) {
+        context.let {
```

Comment:
Why use let here? Could you kill the context param and use the class context with `context?.let` instead? This addresses my previous context related comment also.

Reply:
Yes! Not sure why I had that parameter but agreed is not necessary!